src/core/qgsexpressioncontext.h

```diff
@@ -732,6 +732,8 @@ class CORE_EXPORT QgsExpressionContext
 
   private:
 
+    static const QgsFeature missingFeature;
```

Comment:
Can this be moved to a local static to hide this implementation detail from the public headers?
And in either case, please prefix with an `s`.

Reply:
I removed the static member 8548eab